.github/workflows/release.yml

```diff
@@ -60,12 +121,34 @@ jobs:
         env:
           GORELEASER_KEY: ${{ secrets.GORELEASER_KEY }}
 
-      - name: Version
-        run: |
-          ./scripts/version.js ${{ inputs.increment }}
-          cat version.txt
+      - name: Download Cross-compiled Artifacts
```

Comment:
It would be good to include some code comments in this workflow about why we're doing this. You've explained it a couple times already and I'm still hazy, tbh 😓